application/src/main/java/org/cardanofoundation/ledgersync/dto/healthcheck/HealthStatus.java

```diff
@@ -0,0 +1,17 @@
+package org.cardanofoundation.ledgersync.dto.healthcheck;
+
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class HealthStatus {
+    Boolean isHealthy;
+    String message;
```

Comment:
Should we keep another field for message code?

For example:
message_code = READY_TO_SERVE
messge_desc = "Data is ready to serve"